### ISSUE TITLE
Screen reader doesn't tell when the button is pressed

### ISSUE THREAD

**Issue by charlag** [tutao/tutanota#1608]
We should use `aria-pressed` for that.

## Test Notes
Test with desktop & mobile (iOS & Android) screen readers
- [ ] The buttons works correctly with & without screen readers
  - [ ] The PayPal button in the payment step of the subscription dialog
  - [ ] The search button (Looking glass icon) in the search bar
  - [ ] The 'Show all features' button in the plan table on the subscription dialog
  - [ ] The about button in the settings view
  - [ ] The show attachments expander in the mail viewer
  - [ ] The 'Add folder' button in the mail view sidebar
  - [ ] The select button in the Unlock method dialog
  - [ ] The arrow buttons in the date picker
  - [ ] The arrow buttons in the sidebar calendar
  - [ ] The offline indicator in the toolbar
  - [ ] The today button in the calendar
  - [ ] The mail recipients in the mail editor
- [ ] The following other components reads correctly with screen readers:
  - [ ] The toggle buttons read their labels correctly
  - [ ] The breadcrumb in the subscription & sign up dialogs
  - [ ] The calendar checkboxes in the calendar view sidebar reads correctly
  - [ ] The calendar view selector segment control reads correctly
- [ ] The tab indexing is the same as master

**Comment by charlag** [comment]
All the issues I've found are not new:
 - TalkBack does not recognize "more" button in the top of email
 - VoiceOver does not top of the email as an expander
 - our dropdowns cannot be closed without selecting an action with VoiceOver on iOS
 - focus is not correctly trapped for dropdowns and also some dialogs (like credential encryption method)
 - side menu does not trap the focus correctly
 - VoiceOver on iOS does not recognize the mail list as a list it seems
